setup.cfg

```diff
@@ -9,7 +9,7 @@ force_grid_wrap = 0
 use_parentheses = true
 skip_glob = .eggs*,.venv
 extra_standard_library = hashlib, logging, mimetype, typing, os, sys, abc, io, json, datetime, subprocess, tempfile, shutil, argparse
-known_third_party =PIL,PyPDF2,cairosvg,ffmpeg,filelock,magic,pdf2image,pyexifinfo,pytablewriter,pytest,scribus,wand,xvfbwrapper
+known_third_party =PIL,PyPDF2,filelock,magic,pdf2image,pyexifinfo,pytablewriter,pytest,scribus,wand
```

Comment:
should this value change? Even if those dependencies are now optional they are still third party.

Reply:
done by removing this,  isort>5 doesn't need anymore theses informations.